method/BANKSY/config/config_1.json

```diff
@@ -1 +1 @@
-{"method": "leiden", "lambda": 0.2, "k_geom": 18, "npcs": 20, "use_agf": true}
\ No newline at end of file
+{"desc": "GitHub", "method": "leiden", "lambda": 0.2, "k_geom": 18, "npcs": 20, "use_agf": true}
```

Comment:
Hi sorry for the delay! Everything looks good so far! I need to pick up the conversation a bit but could you remind me what the desc is for?

If it's for HVG and not used in the code atm, maybe we can change `config_2` to sth like `config_2_unused` atm to notify people?

Reply:
The desc is simply to briefly describe the config. Config 1 and 2 differ by "use_agf", which impacts performance.